### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by HechtiDerLachs**
These are some fixes I came up with today while working on other problems. 

@afkafkafk13 : Can you have a quick look at the changes for the space germ functionality?

### CODE REVIEW

File: src/Rings/mpolyquo-localizations.jl [hunk 1]
```diff
@@ -177,7 +177,7 @@ function modulus(L::MPolyQuoLocalizedRing)
 end
 
 ### for compatibility -- also provide modulus in the trivial case
-modulus(R::MPAnyNonQuoRing)=ideal(R,[zero(R)])
+modulus(R::MPAnyNonQuoRing)=ideal(R, elem_type(R)[])
```
Comment (afkafkafk13) [inline src/Rings/mpolyquo-localizations.jl:180]: Since empty lists work now, this is fine with me.

File: experimental/Schemes/SpaceGerms.jl [hunk 2]
```diff
@@ -303,3 +322,14 @@ function singular_locus(X::AbsSpaceGerm)
   Sgerm = SpaceGerm(S)
   return Sgerm, ClosedEmbedding(SpecMor(Sgerm, X, pullback(inc), check=false), image_ideal(inc), check=false)
 end
+
+function point(X::SpaceGerm{<:Field, <:AbsLocalizedRing{<:Ring, <:RingElem, <:MPolyComplementOfKPointIdeal}})
```
Comment (afkafkafk13) [inline experimental/Schemes/SpaceGerms.jl]: Can we move this instance of point up to the two other signatures for point (around line 70 of the same file)?

**Review by afkafkafk13 — COMMENTED** [review]
Looks good to me. 
Just one thing: Please make sure that the signatures, you now added, end up close to the respective existing methods in the file.

File: experimental/Schemes/SpaceGerms.jl (hunk 2 above)
Comment (HechtiDerLachs) [reply experimental/Schemes/SpaceGerms.jl]: I just saw that the method was already there. I just didn't find it earlier.

**Review by afkafkafk13 — APPROVED** [review]

**Comment by simonbrandhorst** [comment]
Could you please add a few tests?

**Pull request merged**